### PR TITLE
PR: Added Cart design

### PR DESCRIPTION
**Pull request by Jayesh3636**
Added cart design and also made it available in product and wishlist page.
Please review it and let me know the changes!
TIA~!

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]

**Review by SukanyaSen530 — COMMENTED** [review]
The UI is amazing :)

File: pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  justify-content: space-between;
+}
+
+.cart .card-horizontal-type-1:hover {
+  color: inherit;
+  background-color: inherit;
+}
+
+.sidepanel {
+
```
Comment (khushijohri2001) [inline pages/cart/cart.css]: please remove this extra line here

File: pages/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+    position: absolute;
+    z-index: 1;
+    height: 68rem;
+    top: 9.5vh;
+    right: -4rem;
+    overflow-x: hidden;
+    overflow-y: scroll;
+    transition: 0.5s;
+    padding: .5rem 0;
+    color: #484955;
```
Comment (khushijohri2001) [inline pages/cart/cart.css]: use a var instead of direct color code

File: pages/cart/cart.css [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+
+.btn .material-icons{
+  margin: 0px;
+  padding: 0;
+}
+
+.minus-icon,
+.plus-icon {
+  align-items: center;
+  /*border: 2px solid #eee;
```
Comment (khushijohri2001) [inline pages/cart/cart.css]: remove unwanted comments

**Review by khushijohri2001 — APPROVED** [review]

File: pages/cart/cart.html [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="https://darkbucket.netlify.app/components/third-party.css" />
+    <link rel="stylesheet" href="./cart.css" />
+    <link rel="stylesheet" href="../../reset.css" />
+    <link rel="stylesheet" href="../../index.css" />
+    <link rel="icon" href="/images/icon.png" />
+    <title>Document</title>
```
Comment (pavan6115) [inline pages/cart/cart.html]: Change the default title to Echo | Cart
Something like this

**Review by pavan6115 — APPROVED** [review]

File: pages/cart/cart.css [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+  padding: .8rem .8rem .8rem 3.2rem;
+  text-decoration: none;
+  font-size: 2.5rem;
+  color: #818181;
+  display: block;
+  transition: 0.3s;
+}
+
+.sidepanel a:hover {
+  color: #f1f1f1;
```
Comment (i-am-a-codegirl) [inline pages/cart/cart.css]: I prefer using var instead of hex code

File: pages/cart/cart.css [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+.cart-amount-details {
+  border: 1px solid #e0e0e0;
+  padding: 2rem;
+  position: relative;
+  margin-bottom: 2rem;
+}
+
+.summary-divider {
+  background-color: #eee;
+  width: 100%;
```
Comment (i-am-a-codegirl) [inline pages/cart/cart.css:239]: same here user var for colors

File: pages/cart/cart.css [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+  text-decoration: none;
+  border-radius: 0.48rem;
+  border: none;
+  cursor: pointer;
+  color: rgb(255, 254, 254);
+  display: inline-block !important;
+  font-family: "Karla", sans-serif !important;
+  font-size: 1.2rem !important;
+  font-weight: 700;
+  letter-spacing: 0.2em;
```
Comment (i-am-a-codegirl) [inline pages/cart/cart.css]: use px for anything less than 10px

File: pages/cart/cart.css [hunk 8]
```diff
@@ -0,0 +1,280 @@
...
+  margin-top: 0 !important;
+  padding-left: 2rem !important;
+  padding-right: 2rem !important;
```
Comment (i-am-a-codegirl) [inline pages/cart/cart.css]: Vikram sir said use specificity if you want these properties to be applied, never use   !important . I would suggest please remove !important and add specificity

**Review by i-am-a-codegirl — APPROVED** [review]
Code looks clean, however avoid using !important as much as possible :) 
Otherwise code looks good to merge

File: pages/cart/cart.css (hunk 5 above)
Comment (Jayesh3636) [reply pages/cart/cart.css]: okay!

File: pages/cart/cart.css (hunk 6 above)
Comment (Jayesh3636) [reply pages/cart/cart.css:239]: done!

File: pages/cart/cart.css (hunk 7 above)
Comment (Jayesh3636) [reply pages/cart/cart.css]: yes!

File: pages/cart/cart.css (hunk 8 above)
Comment (Jayesh3636) [reply pages/cart/cart.css]: I can't right now will change it later

File: pages/cart/cart.html (hunk 4 above)
Comment (Jayesh3636) [reply pages/cart/cart.html]: done

File: pages/cart/cart.css (hunk 2 above)
Comment (Jayesh3636) [reply pages/cart/cart.css]: done

File: pages/cart/cart.css (hunk 3 above)
Comment (Jayesh3636) [reply pages/cart/cart.css]: done

File: pages/cart/cart.css (hunk 1 above)
Comment (Jayesh3636) [reply pages/cart/cart.css]: done

**Comment by Jayesh3636** [comment]
Thanks for reviewing it Guys!

**Pull request merged**